StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1018CodeFixProvider.cs

```diff
@@ -48,7 +48,12 @@ public override FixAllProvider GetFixAllProvider()
                     continue;
                 }
 
-                var nullableType = (NullableTypeSyntax)syntaxRoot.FindNode(diagnostic.Location.SourceSpan);
+                var nullableType = syntaxRoot.FindNode(diagnostic.Location.SourceSpan, getInnermostNodeForTie: true) as NullableTypeSyntax;
```

Comment:
:memo: Failing to include `getInnerMostNodeForTie: true` was the reason we were getting an exception here. The outer node at the location for this syntax error is an `IncompleteMemberSyntax`.